Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/VideoGames.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include<iostream>
+#include <string>
+
+class FVideoGame
+{
+	private:
+		std::string GameName;
+		std::string	StudioName;
```

Comment:
Careful with the format. It's a bit inconsistent, I see that there's an indentation between the string declaration and StudioName, why is that.

A useful shortcut for formatting is `ctrl + k, ctrl + d`. This will format your whole file, because I'm seeing that the formatting is very inconsistent in this review.